test/unit/require.spec.ts

```diff
@@ -33,4 +33,14 @@ export class RequireTests {
         }
     }
 
+    @Test("modules do not have their require paths adjusted")
+    public testRequireModule(): void {
+        const regex = /require\("(.*?)"\)/;
+        const lua = util.transpileString(`
+            import * as fake from "fake";
+            declare module "fake" {}
```

Comment:
Does this test do anything? You didn't decorate your module with the decorator. (Also the name is misleading)

Reply:
Nope it did not. It doesn't look like you can declare a module and import it in the same file. I've made a change to the StringCompilerProgram to allow multiple files to be considered during compilation so I can put through two files to try out this decorator.